File: app/src/main/java/com/tarlic/RPGDiceRoller/LogItem.java
package com.tarlic.RPGDiceRoller;

import java.util.Date;

public class LogItem {
    public String text;
    public Date date;
    public LogItem(){
        super();
    }
    
    public LogItem(String text, Date date) {
        super();
        this.text= text;
        this.date= date;
    }
	
	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}

	public Date getDate() {
		return date;
	}

	public void setDate(Date date) {
		this.date = date;
	}
}

File: app/src/main/java/com/tarlic/RPGDiceRoller/LogItem.kt
package com.tarlic.RPGDiceRoller

import java.util.*

class LogItem {
    var text: String? = null
    var date: Date? = null

    constructor() : super() {}
    constructor(text: String?, date: Date?) : super() {
        this.text = text
        this.date = date
    }
}